src/__init__.py

```diff
@@ -115,13 +115,11 @@ def create_items(self):
                 new_item = self.create_item(name)
                 pool.append(new_item)
 
-            if item.get("early") and item.get("local"): # both
-                self.multiworld.local_early_items[self.player][name] = item_count
```

Comment:
It sounds like local early items is still a thing on the multiworld. Did we want to keep this? @nicopop

Reply:
well it work the same witouth

Reply:
but as a bonus the local items appear in the spoiler log

Reply:
we might need to decide if we prioritize the json over the player yaml, 
that way we could check if the item is in the wrong local/non_local/both and remove it from there or not

Reply:
Something like
```
            if item.get("local", None) != None: # only local
                if item.get("local"):
                    if name not in self.options.local_items.value:
                        self.options.local_items.value.add(name)
                else:
                    if name not in self.options.non_local_items.value:
                        self.options.local_items.value.add(name)
```

Reply:
On principle, YAML will always take precedence over the JSON when they can have contradicting settings. This allows players to customize their easy or painful seed as much as they want. 😃 And if the YAML doesn't generate, they can fix their YAML.